lib/jorb/backend.ex

```diff
@@ -0,0 +1,20 @@
+defmodule Jorb.Backend do
```

Comment:
I just noticed you use both `Jorb.Backend` and `Jorb.Backends`.  This is going to confuse people (definitely me).

Reply:
My idea here was that `Jorb.Backend` was the behavior and `Jorb.Backends` contained the implementations, but I think you're right, it probably should be just `Backend`